cmd/cmp/main.go

```diff
@@ -20,11 +17,15 @@ var (
 )
 
 func Run(_ *cobra.Command, args []string) {
+	returnCode := CompareRun(args)
+	os.Exit(returnCode)
+}
+
+func CompareRun(args []string) int {
 	var (
 		err       error
 		fileIdMap map[string]string
 	)
-
 	fileIdRegex, err = regexp.Compile(FileIdRegexStr)
 	if err != nil {
 		log.Fatal().Err(err).Msg("failed to compile file ID regex")
```

Comment:
Can you add this after line 34 (`buildSidePath, probeSidePath := args[0], args[1]`):
```go
utils.ExpandHomeDirectory(&buildSidePath)
utils.ExpandHomeDirectory(&probeSidePath)
```

This is to handle the "~" in path, which should be translated to the home folder path.